argo/workflow/release/region-available.yaml

```diff
@@ -9,8 +9,18 @@ spec:
         value: tipocket-region-available
       - name: purge
         value: "true"
+      - name: hub
+        value: "docker.io"
```

Comment:
Do we need to change hub to our internal hub?

Reply:
Updated